e2e_testing/run_model_forward.py

```diff
@@ -0,0 +1,82 @@
+import time
```

Comment:
Is this exercised by CI? The Llama and Mixtral models are tested on (CPU) CI with a small configuration via `pytest`. See e.g. https://github.com/AI-Hypercomputer/torchprime/blob/main/torchprime/torch_xla_models/tests/test_llama.py

I think in order to test this in CI we'd need a file with the `test_` prefix.

Reply:
the `run_model_forward` script is meant to run as a CLI script. I added a CI entry to test it directly.

Reply:
Hmm, not sure why the E2E test doesn't pick up this file.

FWIW, we've been using E2E test to mean "exercise distributed training workflows", so actually the VM the E2E test runs on only submits jobs.

Would it be possible to write this `run_model_forward.py` as a parameterized pytest test? I think if you put it in `torchprime/tests` and give it a name like `test_run_model.py`, then 

https://github.com/AI-Hypercomputer/torchprime/blob/main/.github/workflows/cpu_test.yml#L50

will pick it up automatically.

Reply:
done. PTAL